nimbleclient/v1/restclient.py

```diff
@@ -203,30 +228,40 @@ def get(self, endpoint, **params):
             paginated_data.extend(response.json()["data"])
             total_rows = response.json()['totalRows']
             retrieved_rows = len(paginated_data)
-            #current_end_row = response.json()['endRow']
+            # current_end_row = response.json()['endRow']
 
             if 'startRow' in params and 'endRow' in params:
                 requested_rows = params['endRow'] - params['startRow']
             elif 'startRow' not in params and 'endRow' not in params:
-                requested_rows = total_rows # Get all available records
+                requested_rows = total_rows  # Get all available records
             else:
                 if 'startRow' in params:
                     requested_rows = total_rows - params['startRow']
                 else:
                     requested_rows = params['endRow']
-            #print("Total records requested:", requested_rows, ", Retrieved records:", retrieved_rows)
+
+            logging.debug("Total records requested: %d, Retrieved records: %d", requested_rows, retrieved_rows)
 
             pending_rows = requested_rows-retrieved_rows
             params.clear()
```

Comment:
I don't think `clear()` is a function that you can call on a `dict`. If you just want a fresh dict, just make `params = {}`

Reply:
Fixed